APIWrapper/Objects/ScheduleHourContent.cs

```diff
@@ -0,0 +1,67 @@
+using APIWrapper.Enums;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace APIWrapper.Objects
```

Comment:
We have to discuss the structure at next meeting, I added it to program already..